enterprise/app/trends/heatmap.tsx

```diff
@@ -220,12 +227,13 @@ class HeatmapComponentInternal extends React.Component<HeatmapProps, State> {
     if (!data) {
       return;
     }
+    document.addEventListener("mousemove", this.onMouseMove);
     document.addEventListener("mouseup", (e) => this.onMouseUp(e), { once: true });
     this.pendingClick = [data, data];
     this.setState({ selectionToRender: this.pendingClick });
   }
 
-  onMouseMove(e: React.MouseEvent<SVGSVGElement, MouseEvent>) {
+  onMouseMove = (e: MouseEvent) => {
```

Comment:
heh, this syntax kinda grosses me out a bit even though i understand it's perfectly valid.  there's part of me that would almost prefer defining the field as `(e) => this.onMouseMove(e)` and keeping the function defined as-is, but i realize this is just a quirk of mine.  not sure why i'm still typing.